### PR TITLE
[RHPAM-2279/RHDM-1057] update amq and datagrid images

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: Filippe Spolti <fspolti@redhat.com>

### CODE REVIEW

**Comment by tchughesiv** [comment]
@spolti which product release is this targeted for?

**Comment by spolti** [comment]
@tchughesiv 7.6 I believe, @errantepiphany Can you please confirm?

**Comment by errantepiphany** [comment]
@tchughesiv @spolti This is just for product 7.6.0. According to our meeting notes from October 15, "Decision is to keep with current version we’ve been using for 7.5.0.GA"

File: pkg/apis/app/v2/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // +build !ignore_autogenerated
 
-// Code generated by deepcopy-gen. DO NOT EDIT.
+// Code generated by operator-sdk. DO NOT EDIT.
```
Comment (tchughesiv) [inline pkg/apis/app/v2/zz_generated.deepcopy.go]: This appears when using newer sdk version. Can you please ensure version 0.7.0 was used and fix commit if necessary?

**Pull request merged**